Resources/Prototypes/Roles/Jobs/Civilian/lawyer.yml

```
@@ -25,5 +25,5 @@
     - BriefcaseBrownFilled
   storage:
     back:
-    - RubberStampLawyer
+    #- RubberStampLawyer  Goob - Removed to be placed in loadout
```

yamllint (1.35.1)

[warning] missing starting space in comment

(comments)